app/cdap/text/text-en.yaml

```diff
@@ -3436,4 +3436,10 @@ features:
       gridTable:
         title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+    GridPage:
+      directivePanel:
+        dollar: $
```

Comment:
dont add $ in i18n, we can directly use this symbol in code

Reply:
Done